apps/challenges/migrations/0019_added_leaderboard_data_table.py

```diff
@@ -10,7 +10,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('jobs', '0004_submission_output'),
+        ('jobs', '0001_squashed_0005_upload_unique_random_filename'),
```

Comment:
@varunagrawal This is not the part of this PR. Can you please open a separate PR for this?